terraform/modules/happy-iam-service-account-eks/variables.tf

```diff
@@ -49,6 +49,11 @@ variable "max_session_duration" {
   default     = 3600
 }
 
+variable "aws_iam_policies_json" {
```

Comment:
I would have changed the variable type of `var.aws_iam_policy_json`, but  I didn't want to make a breaking change.